Tests/js/models/assets/ez-contentmodel-tests.js

```diff
@@ -639,6 +639,72 @@ YUI.add('ez-contentmodel-tests', function (Y) {
         },
     });
 
+    deleteContent = new Y.Test.Case({
+        name: "eZ Content Model delete content tests",
+
+        setUp: function() {
+            this.model = new Y.eZ.Content();
+            this.contentId = 'Pele';
+
+            this.capi = new Mock();
+            this.contentService = new Mock();
+
+            Y.Mock.expect(this.capi, {
+                method: 'getContentService',
+                returns: this.contentService
+            });
+
+            Y.Mock.expect(this.model, {
+                method: 'get',
+                args: ['id'],
+                returns: this.contentId
+            });
+        },
+
+        tearDown: function () {
+            this.model.destroy();
+            delete this.model;
+            delete this.capi;
+            delete this.contentService;
+        },
+
+        "Should delete the content": function () {
+            var options = {api: this.capi},
+                content = this.model;
+
+            Y.Mock.expect(this.contentService, {
+                method: 'deleteContent',
+                args: [this.contentId, Y.Mock.Value.Function],
```

Comment:
you don't need 2 tests for this method. You only have to check that the delete `callback` is passed to the content service `deleteContent` method. you can write something like:

```js
var options = {api: this.capi},
    callback = function () {},
    content = this.model;

Y.Mock.expect(this.contentService, {
    method: 'deleteContent',
    args: [this.contentId, callback],
});
content.delete(options, callback);
```